src/connection.hpp

```diff
@@ -96,6 +96,12 @@ class Connection
   /// The incoming request.
   std::unique_ptr<Request> request_;
 
+  // string containing http resonse
```

Comment:
reaaaaaaaally nitpicky but the style of this comment doesn't match the other ones in the file lol